webapp/src/components/jira_issue_selector/jira_issue_selector.jsx

```diff
@@ -27,11 +26,9 @@ export default class JiraIssueSelector extends Component {
     };
 
     searchIssues = (text) => {
-        const projectSearchTerm = this.props.currentProject ? 'project=' + this.props.currentProject : '';
         const textEncoded = encodeURIComponent(text.trim().replace(/"/g, '\\"'));
         const textSearchTerm = (textEncoded.length > 0) ? 'text ~ "' + textEncoded + '*"' : '';
-        const combinedTerms = (projectSearchTerm.length > 0 && textSearchTerm.length > 0) ? projectSearchTerm + ' AND ' + textSearchTerm : projectSearchTerm + textSearchTerm;
-        const finalQuery = combinedTerms + ' ' + searchDefaults;
+        const finalQuery = textSearchTerm + ' ' + searchDefaults;
```

Comment:
Not sure I fully understand this one, but if so... nice! (simplification)

Reply:
@levb since we're no longer using project to narrow down the issue search, no need to include it in the jql query. = free simplification. :)